### DISCUSSION TITLE
Comparing three  hex files

### DISCUSSION THREAD

**Discussion by DrSmoleTT** [WinMerge/winmerge#2162] [Q&A]
I'm comparing three hex files and I need to find differences based on some conditions. For example, select only those fragments that are the same in two files and different in the third. How can I set my comparison conditions?

**Comment by sdottaka** [comment]
Unfortunately, Binary Compare does not have a feature like Synchronization Points like Text Compare.

**Comment by DrSmoleTT** [comment]
Oh this is so sad...
Perhaps there is an opportunity to automate my actions? Is it possible to create scripts for specific actions? Assign hotkeys?
Even the patch doesn't allow it to be done for binary files...
